cmd/client/app/generate.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright © 2021 Dan Lorenc <lorenc.d@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/sigstore/fulcio/pkg/log"
+
+	"github.com/pkg/errors"
+	"github.com/sigstore/fulcio/pkg/ca"
+	"github.com/sigstore/fulcio/pkg/ctl"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	privatecapb "google.golang.org/genproto/googleapis/cloud/security/privateca/v1beta1"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+var (
+	pubKeyFile string
+	uri        string
+)
+
+var generateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate a fulcio-signed certificate",
+	Long:  `Generate is used to create a fulcio-signed certificate without OIDC`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := generateCert(context.Background()); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+	},
+}
+
+func generateCert(ctx context.Context) error {
+	pemBytes, err := ioutil.ReadFile(filepath.Clean(pubKeyFile))
+	if err != nil {
+		return errors.Wrap(err, "reading pub key file")
+	}
+	parent := viper.GetString("gcp-private-ca-parent")
+	req := cert(parent, uri, pemBytes)
+	resp, err := ca.Client().CreateCertificate(ctx, req)
+	if err != nil {
+		return errors.Wrap(err, "creating cert")
+	}
+
+	// Submit to CTL
+	log.Logger.Info("Submitting cert to CT log")
```

Comment:
Just thought, i wonder if we might want this to be optional. I can certainly see why in most cases a CTL entry would be desirable, but there may be cases where someone just wants a cert and would like to skip entry. I say that assuming this is intended as an offline type tool for an admin to use.